portability-spi-transfer/src/main/java/org/dataportabilityproject/spi/transfer/provider/Importer.java

```diff
@@ -21,6 +21,9 @@
 /** Imports data into a destination service. */
 public interface Importer<A extends AuthData, T extends DataModel> {
 
+  /** The key associated with this service. */
+  String getServiceId();
```

Comment:
Per off line conversation, let's revert this

Reply:
Done